src/helpers/library.ts

```diff
@@ -249,43 +249,45 @@ type LaunchOptions = {
   t: TFunction<'gamepage'>
   launchArguments?: string
   runner: Runner
+  hasUpdate: boolean
 }
 
 const launch = async ({
   appName,
   t,
   launchArguments,
-  runner
-}: LaunchOptions): Promise<void> =>
-  ipcRenderer
+  runner,
+  hasUpdate
+}: LaunchOptions): Promise<void> => {
+  if (hasUpdate) {
+    const args = {
+      buttons: [t('gamepage:box.yes'), t('box.no')],
+      message: t('gamepage:box.update.message'),
+      title: t('gamepage:box.update.title')
+    }
+
+    const { response } = await ipcRenderer.invoke('openMessageBox', args)
+
+    if (response === 0) {
+      return updateGame(appName, runner)
+    }
+
+    return ipcRenderer.invoke('launch', {
+      appName,
+      runner,
+      launchArguments: runner === 'legendary' ? '--skip-version-check' : ''
```

Comment:
I think this should be done in `launch` directly instead of appending it here

Reply:
you mean the check? because the backend needs to receive the argument from here so then it can pass it to legendary.
Honestly, I just don't know how GOGDL react to this cli, otherwise we could pass it to both.

Reply:
Ok, gogdl ignores the flag so I can remove that check